Slack.php

```diff
@@ -340,10 +392,11 @@ function notify($msg, $webhook, $channel, $attachment = FALSE) {
         curl_setopt($ch, CURLOPT_CONNECTTIMEOUT, 5);
         curl_setopt($ch, CURLOPT_TIMEOUT, 5);
         $result = curl_exec($ch);
-        if ($result !== 'ok') {
+        //ignore error to avoid blocking mantis if slack is down
```

Comment:
I'll remove this unrelated change - let's discuss it in a separate issue if needed.